src/index.ts

```diff
@@ -1,65 +1,85 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
-/* eslint-disable @typescript-eslint/no-unsafe-call */
-/* eslint-disable @typescript-eslint/no-unsafe-member-access */
-/* eslint-disable @typescript-eslint/no-var-requires */
 /**
  * @Author: Rostislav Simonik <rostislav.simonik@technologystudio.sk>
  * @Date: 2020-06-10T16:06:00+02:00
  * @Copyright: Technology Studio
 **/
 
-import type { Linter } from 'eslint'
 import globals from 'globals'
+import {
+  configs as typescriptEslintConfigs,
+  parser,
+} from 'typescript-eslint'
+import type { TSESLint } from '@typescript-eslint/utils'
+import eslintComments from '@eslint-community/eslint-plugin-eslint-comments/configs'
+import { flatConfigs as eslintImportFlatConfigs } from 'eslint-plugin-import'
+import eslintImportRecommendedFlatConfig from 'eslint-plugin-import/config/flat/recommended'
+import stylistic from '@stylistic/eslint-plugin'
+import jestPlugin from 'eslint-plugin-jest'
+import loveConfig from 'eslint-config-love'
```

Comment:
let's unify naming convention to follow the import name if it's scoped and it's eslint-plugin or eslint-config let's add scope as suffix